packages/cloud-cognitive/src/components/SidePanel/SidePanel.js

```diff
@@ -91,25 +91,55 @@ export let SidePanel = React.forwardRef(
       }
     }, [selectorPrimaryFocus, open, animationComplete]);
 
-    // Title animaton
+    // Title and subtitle animaton
     useEffect(() => {
       if (open && animateTitle && animationComplete) {
         const sidePanelOuter = document.querySelector(`#${blockClass}-outer`);
+        const sidePanelSutitleElement = document.querySelector(
```

Comment:
Sorry to be the pedant in the room :-) but can we fix this to `sidePanelSubtitleElement`?! I haven't done it as a suggestion here because it occurs in a number of places.

Reply:
Doh, sorry about that. Yes